doc improvement: LwM2M client sample document update (#12733)
nrfconnect/sdk-nrf: doc/nrf/includes/lwm2m_common_server_setup.txt

         .. coiote_start

>          If bootstrap is enabled please skip following steps.
>          Bootstrap server creates mangamenet device automatically during the bootstrap.

Reviewer: ```suggestion
         If bootstrap is enabled, you can skip the following steps.
         The bootstrap server creates the management device automatically during the bootstrap.
```
